java/fury-core/src/main/java/org/apache/fury/memory/MemoryBuffer.java

```diff
@@ -90,6 +92,7 @@ public final class MemoryBuffer {
   private int size;
```

Comment:
Since the `size` field did not represent actual buffer _size_, maybe it worth renaming it? WDYT @chaokunyang?

Reply:
Do you have any suggestions? I think size is still fine, bytearrayOutputStream use count, ByteBuffer use capacity

Reply:
Make sense, I''ll update it later

Reply:
Change this make this PR a little messy, how about renaming it in another PR.